docs/how-tos/how-to-use-a-custom-promotion-adjuster.mdx

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 7
+---
+
+# How to use a custom promotion adjuster
+
+Every time an order is updated, its eligible promotions might
+not be valid anymore; for example, this could happen if a promotion is
+tight to a specific order total threshold, and this value changes within
+the update. In the same way, changing the order could make it eligible
+for new promotions.
+
+In this guide, we'll see how to create a custom promotion adjuster to
+use our custom logic that handles promotion adjustments every time an
+order is updated.
+
+In Solidus, this behavior is encapsulated in the
+[`Spree::Promotion::OrderAdjustmentsRecalculator`][order_adjustments_recalculator],
+class, which is called every time the order is updated. This class iterates over all
+existing promotion adjustments and recalculates their amount and eligibility.
+
+:::caution
+
+Please, keep in mind that promotion adjustments are handled before taxes are
+calculated, such that taxes always respect promotions.
+
+:::
+
+We'll now see how to use our own class instead of the one provided
+by Solidus.
+
+First of all, let's declare which promotion adjuster class we want to
+use, with the corresponding configuration:
+
+```ruby title="config/initializers/spree.rb"
+Spree.config do |config|
+  # ...
+  config.promotion_adjuster_class = 'MyStore::PromotionAdjuster'
+end
+```
+
+Next, let's create our custom class, which responds to the same interface
```

Comment:
```suggestion
Lastly, let's create our custom class, which implements the same interface
```

Using `Next` seems to convey it's not the last step. For me, the end was too abrupt otherwise. Another option is to add a paragraph with a short conclusion.

Also a semantic nit.